Assets/AnkrSDK/Examples/Scripts/ERC721Example/ERC721Example.cs

```diff
@@ -1,15 +1,17 @@
 ﻿using System;
 using System.Numerics;
+using AnkrSDK.Core.Data;
 using AnkrSDK.Core.Data.ContractMessages.ERC721;
 using AnkrSDK.Core.Implementation;
 using AnkrSDK.Core.Infrastructure;
 using AnkrSDK.Examples.DTO;
-using AnkrSDK.WalletConnectSharp.Unity;
+using AnkrSDK.UseCases;
+using Nethereum.RPC.Eth.DTOs;
 using UnityEngine;
 
 namespace AnkrSDK.Examples.ERC721Example
 {
-	public class ERC721Example : MonoBehaviour
+	public class ERC721Example : UseCase
```

Comment:
Why did you make them a use case?

Reply:
Because I need to test it, but without UseCase it's very difficult